bootstrap_cfn/fab_tasks.py

```diff
@@ -738,7 +748,14 @@ def cfn_create(test=False):
         # So delete the SSL cert that we uploaded
         if 'ssl' in cfn_config.data:
             iam.delete_ssl_certificate(cfn_config.ssl(), stack_name)
-        abort('Failed to create stack: {0}'.format(stack))
+        # delete the DNS tag
+        logger.info("Deleting DNS tag for '{stack} from '{zone}'...".format(stack=stack_id, zone=zone_name))
+        try:
+            print green("Removing tag stack from DNS...")
+            r53_conn.delete_txt_record(zone_name, zone_id, txt_tag_record)
+        except boto.route53.exception.DNSServerError:
+            print red("Failed to remove DNS tag")
+        abort(red('Failed to create stack: {0}. You may want to delete it manually.'.format(stack)))
```

Comment:
Retaining the rolledback stack is current behaviour so I decided not to change it in this patch. I've a slight preference for *not* automatically deleting a rolledback stack but am open to discussion. But even if so, we should probably do it as a separate piece of work.